beginend.el

```diff
@@ -2,7 +2,7 @@
 
 ;; Copyright (C) 2015-2017 Damien Cassou
 
-;; Author: Damien Cassou <damien@cassou.me>
+;; Authors: Damien Cassou <damien@cassou.me> and Matus Goljer <matus.goljer@gmail.com>
```

Comment:
If there are multiple authors they should be listed on separate lines, see `Author` https://www.gnu.org/software/emacs/manual/html_node/elisp/Library-Headers.html